Simplenote/Classes/SPNoteEditorViewController.m

```diff
@@ -117,12 +116,11 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
 
-    [self setupNavigationController];
     [self highlightSearchResultsIfNeeded];
     [self startListeningToKeyboardNotifications];
-
     [self refreshNavigationBarButtons];
-
+    [self configureNavigationControllerToolbar];
+    [self setupNavigationController];
```

Comment:
**Nipticky!!**

This API was already there (not really new code!!), buuut for consistency's sake, mind renaming `setupNavigationController` > `configureNavigationController` so that all the things look the same?

**Thank you!!**